### PR TITLE
fix(script): migration for betanet and testnet

### PR DESCRIPTION
**Pull request by ailisp**
There's trivial error in migration script. It wasn't caught because ci state migration is tested on `neard init` genesis, which doesn't have Data records. Also fix validator section inconsistent issue due to old state viewer bug, it's needed for betanet and testnet upgrade.

Test Plan
------------
Run script again diff with yesterday's manual fix

### CODE REVIEW

File: scripts/migrations/9-state-record-data.py [hunk 1]
```diff
@@ -18,7 +20,7 @@
 
 config = json.load(open(os.path.join(home, 'output.json')), object_pairs_hook=OrderedDict)
 
-assert config['protocol_version'] == 8
+# assert config['protocol_version'] == 8
```
Comment (bowenwang1996) [inline scripts/migrations/9-state-record-data.py:23]: ?

Comment (ailisp) [reply scripts/migrations/9-state-record-data.py:23]: forget to uncomment

**Comment by ailisp** [comment]
@evgenykuzyakov my faul, state-migration test doesn't really test actual betanet/testnet genesis, I think I can make them test with them

**Comment by bowenwang1996** [comment]
@ailisp I created an issue #2486 for this.

**Pull request merged**